modules/local/torch_tune.nf

```diff
@@ -3,30 +3,31 @@ process TORCH_TUNE {
 
     tag "${model} - ${combination_key}"
     label 'process_high'
-    container "mathysgrapotte/stimulus-py:latest"
+    // TODO: push image to nf-core quay.io
+    container "docker.io/mathysgrapotte/stimulus-py:latest"
 
     input:
     tuple val(combination_key), val(split_transform_key), path(ray_tune_config), path(model), path(data_csv), path(experiment_config), path(initial_weights)
 
     output:
     tuple val(split_transform_key),
-          val(combination_key),
-          path(data_csv),
-          path(experiment_config),
-          path("*-config.json"),
-          path("*-model.safetensors"),
-          path("*-optimizer.pt"),
-          path("*-metrics.csv"),
-          path(initial_weights),
-          emit: tune_specs
+        val(combination_key),
```

Comment:
Just a note that if you plan to put this module to nf-core/modules it's better to have the output tuple in one single line. We use regular expressions to parse it and lint the main.nf and meta.yml, so I don't think this will work for an nf-core module.